common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -46,23 +49,53 @@ export default class AccountInfo extends React.Component<Props, State> {
     });
   };
 
+  public displayHWAddress = () => {
+    const { hwType, wallet } = this.props;
+    const { dPath, index } = wallet as any;
+    if (hwType === 'ledger') {
+      ledger.comm_u2f.create_async().then(comm => {
+        new ledger.eth(comm).getAddress_async(dPath + '/' + index, true, false);
+      });
+    } else if (hwType === 'trezor') {
+      TrezorConnect.ethereumGetAddress(
+        dPath,
+        (result, error) => {
+          if (result) {
+            console.log(result);
+          } else {
+            throw new Error(error);
+          }
```

Comment:
You can get rid of these two. We can probably just swallow the error if there was one, it's probably because they disconnected their device. Alternatively you could show a notification.